webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Details/Cards/Virtualization/index.js

```diff
@@ -0,0 +1,75 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {
+  DescriptionList,
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+} from '@patternfly/react-core';
+import { translate as __ } from '../../../../../../common/I18n';
+import { foremanUrl } from '../../../../../../common/helpers';
+import { useAPI } from '../../../../../../common/hooks/API/APIHooks';
+import CardTemplate from '../../../../Templates/CardItem/CardTemplate';
+import VirtVmware from './VirtVmware';
+import VirtOvirt from './VirtOvirt';
+import VirtLibvirt from './VirtLibvirt';
+import VirtEc2 from './VirtEc2';
+import VirtGce from './VirtGce';
+import VirtOpenstack from './VirtOpenstack';
+
+const VirtualizationCard = ({ hostDetails, hostName }) => {
+  const {
+    compute_resource_id: computeResourceId,
+    compute_resource_name: computeResourceName,
+    compute_resource_provider: provider,
+  } = hostDetails;
+  const virtUrl = foremanUrl(`/api/hosts/${hostName}/vm_compute_attributes`);
+  const { response: vm, status } = useAPI('get', virtUrl);
+
+  if (provider === undefined) return null;
```

Comment:
provider can be null which crashes the page
```suggestion
  if (!provider) return null;
```